### DISCUSSION TITLE
Copied comment from stackoverflow

### DISCUSSION THREAD

**Discussion by Novarest** [rrousselGit/provider#839] [Q&A]
**Describe the bug**
In the change_notifier_provider.dart we found a copied comment from stackoverflow from Rémi Rousselet:

/// See [this stackoverflow answer](https://stackoverflow.com/questions/52249578/how-to-deal-with-unwanted-widget-build)
/// which explains in further details why using the `.value` constructor to
/// create values is undesired.

Therefore this file falls under the Creative Commons Attribution Share Alike 4.0 International.

**Expected behavior**
Please rewrite or remove this comment if you do not want the file to be covered by this license

**Comment by rrousselGit** [comment]
What's the issue here?
I both made the question and the answer. I'm not sure why I wouldn't be able to cross-link to it.
